python/bigipconfigdriver.py

```diff
@@ -609,51 +845,60 @@ def _do_reset(self):
                     # Manually create custom profiles;CCCL doesn't yet do this
                     if 'customProfiles' in config['resources']:
                         for profile in config['resources']['customProfiles']:
-                            _create_client_ssl_profile(self._bigip, profile)
+                            _create_client_ssl_profile(
+                                self._cccls[0]._bigip_proxy.mgmt_root(),
+                                profile)
                             customProfiles = True
 
-                    cfg = create_config_kubernetes(self._bigip, config)
-                    if self._bigip.regenerate_config_f5(cfg):
+                    incomplete = 0
+                    for cccl in self._cccls:
+                        cfg = create_config_kubernetes(cccl, config)
+                        try:
+                            incomplete += cccl._apply_config(cfg)
```

Comment:
This is confusing because it is calling the _apply_config() from earlier in this file (because cccl in this case isn't a real cccl, it's our derived extension of cccl).

Erik, I think you said you'd look at changing this to has-a-cccl instead of is-a-cccl